Checklists/README.md

```diff
@@ -14,3 +14,55 @@ The MSTG version element (Dashboard - row 13) in the excel file represent the ve
 Note that due to the continuous updating and restructuring of the MSTG, the links of different versions are not all identical. And changing the mstg version (Dashboard - row 13) may break the links in the excel file.
 
 Newer and older versions of the Excel are released on a regular basis and can be found at [the release page](https://github.com/OWASP/owasp-mstg/releases "Releases").
+
+## Files Hash
+
+|  Filename                                         |  SHA256 Sum                                                      |
+|---------------------------------------------------|------------------------------------------------------------------|
+| Mobile_App_Security_Checklist-English_1.1.2.xlsx  | b1f101bd3ea87e3a7e556cce2c00de925799e6acab06f406fd9f07cf5f990f17 |
+| Mobile_App_Security_Checklist-French_1.1.2.xlsx   | 4ac489734a009f1101aab37cd96925b067e19baead9e4b395a7048ad026844b5 |
+| Mobile_App_Security_Checklist-Japanese_1.1.2.xlsx | a5c636fba4119305197bfa7bc38dbb15b704f3437312f9481896984b16160542 |
+| Mobile_App_Security_Checklist-Spanish_1.1.xlsx    | f561377a9e45e235af0db9bcf483577e55c4cfd5a40a42ea6a9335681add9ccc |
+
```

Comment:
You don't need to do table formatting as such, can you change it please? See the suggestion below:
```suggestion
| Filename | SHA256 Sum |
|---|---|
| Mobile_App_Security_Checklist-English_1.1.2.xlsx | b1f101bd3ea87e3a7e556cce2c00de925799e6acab06f406fd9f07cf5f990f17 |
| Mobile_App_Security_Checklist-French_1.1.2.xlsx | 4ac489734a009f1101aab37cd96925b067e19baead9e4b395a7048ad026844b5 |
| Mobile_App_Security_Checklist-Japanese_1.1.2.xlsx | a5c636fba4119305197bfa7bc38dbb15b704f3437312f9481896984b16160542 |
| Mobile_App_Security_Checklist-Spanish_1.1.xlsx | f561377a9e45e235af0db9bcf483577e55c4cfd5a40a42ea6a9335681add9ccc |
```